src/App.js

```diff
@@ -1,16 +1,42 @@
 import React, { Component } from 'react'
 import { Instructions } from './components'
+import { Counter } from './components'
 import './styles/app.css'
 
 class App extends Component {
+  //  var mylist = ["Ayy", "whats", "good"]
+  //  const mapper = list.map(x : x + "\n")
   render() {
+    let mylist = ['Ayy', 'whats', 'good']
+    mylist = mylist.filter(x => x.length > 2)
+    mylist = mylist.map((x, index) => (index % 2 === 1 ? x.toUpperCase() : x))
+
+    const mapper = mylist.map(x => (
```

Comment:
`mapper` isnt necessarily the best name for this since it sounds like a function that executes some sort of mapping as opposed to the result of something being mapped